server/config.py

```diff
@@ -0,0 +1,47 @@
+import os
+from sqlalchemy.dialects import registry
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+db_base = 'cockroachdb://kavish:erKaCOuWe-zIMxPe@free-tier.gcp-us-central1.cockroachlabs.cloud:26257/second-jaguar' \
+          '-3728.defaultdb?sslmode=verify-full&sslrootcert=root.crt '
+# database_name = 'flask_jwt_auth'
+
+
+class BaseConfig:
+    """Base configuration."""
+    # registry.register("cockroachdb", "cockroachdb.sqlalchemy.dialect",
+    #                   "CockroachDBDialect")
+    SECRET_KEY = os.getenv('SECRET_KEY', 'your_super_secret_key')
+    DEBUG = False
+    BCRYPT_LOG_ROUNDS = 12
+    registry.register("cockroachdb", "cockroachdb.sqlalchemy.dialect",
+                      "CockroachDBDialect")
+    SQLALCHEMY_DATABASE_URI = db_base
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+
+class DevelopmentConfig(BaseConfig):
+    """Development configuration."""
+    registry.register("cockroachdb", "cockroachdb.sqlalchemy.dialect",
+                      "CockroachDBDialect")
+    DEBUG = True
+    # BCRYPT_LOG_ROUNDS = 4
+    SQLALCHEMY_DATABASE_URI = db_base
+
+
+class TestingConfig(BaseConfig):
```

Comment:
This is very important. Why? We were doing testing on the prod database which is really really bad